src/control-plane/backend/lambda/api/store/aws/s3.ts

```diff
@@ -125,3 +100,63 @@ export const isBucketExist = async (region: string, bucket: string) => {
     return false;
   }
 };
+
+export async function putStringToS3(
+  content: string,
+  region: string,
+  bucketName: string,
+  key: string,
+) {
+  const s3Client = new S3Client({
+    ...aws_sdk_client_common_config,
+    region,
+  });
+  await s3Client.send(
+    new PutObjectCommand({
+      Bucket: bucketName,
+      Key: key,
+      Body: content,
+    }),
+  );
+}
+
+export async function readS3ObjectAsString(bucketName: string, key: string): Promise<string | undefined> {
+  try {
+    const s3Client = new S3Client({
+      ...aws_sdk_client_common_config,
```

Comment:
miss `region` for working on opt-in region

Reply:
done